helm/values-prod.yaml

```diff
@@ -174,8 +174,8 @@ microservice-chart:
     SESSION_URL_OUTCOME_SUFFIX: "/ecommerce-fe/esito#clientId={clientId}&transactionId={transactionId}&sessionToken={sessionToken}"
     SESSION_URL_NOTIFICATION_URL: "https://api.platform.pagopa.it/ecommerce/npg/notifications/v1/sessions/{orderId}/outcomes?sessionToken={sessionToken}"
     JWT_NPG_NOTIFICATION_VALIDITY_TIME: "900"
-    REDIRECT_PAYMENT_TYPE_CODE_LIST: "PPAYITR1XXX-RBPB,CHECKOUT-PPAYITR1XXX-RBPP,PPAYITR1XXX-RBPR,RBPS,RPIC,CHECKOUT-BPPIITRRXXX-RBPR,CHECKOUT-BPPIITRRXXX-RBPP,CHECKOUT-BPPIITRRXXX-RBPB,IO-PPAYITR1XXX-RBPP,RICO"
-    REDIRECT_PAYMENT_TYPE_CODE_DESCRIPTION_MAPPING: '{"RBPR":"Poste addebito in conto Retail","RBPB":"Poste addebito in conto Business","RBPP":"Paga con BottonePostePay","RPIC":"Pago in Conto Intesa","RBPS":"SCRIGNO Internet Banking","RICO":"Conto ICONTO"}'
+    REDIRECT_PAYMENT_TYPE_CODE_LIST: "PPAYITR1XXX-RBPB,CHECKOUT-PPAYITR1XXX-RBPP,PPAYITR1XXX-RBPR,RBPS,RPIC,CHECKOUT-BPPIITRRXXX-RBPR,CHECKOUT-BPPIITRRXXX-RBPP,CHECKOUT-BPPIITRRXXX-RBPB,IO-PPAYITR1XXX-RBPP,RICO,KLAR"
```

Comment:
watch out: adding this configuration require that secret have already been upgrade in prod environment before merge and deploy, otherwise service wont start up

Reply:
Thank you @pietro-tota